api/v1beta1/nutanixmachine_types.go

```diff
@@ -144,14 +169,13 @@ type NutanixMachineStatus struct {
 	FailureMessage *string `json:"failureMessage,omitempty"`
 }
 
-//+kubebuilder:object:root=true
-//+kubebuilder:resource:path=nutanixmachines,shortName=nma,scope=Namespaced,categories=cluster-api
-//+kubebuilder:subresource:status
-//+kubebuilder:storageversion
-//+kubebuilder:printcolumn:name="Address",type="string",JSONPath=".status.addresses[0].address",description="The VM address"
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=nutanixmachines,shortName=nma,scope=Namespaced,categories=cluster-api
+// +kubebuilder:subresource:status
+// +kubebuilder:storageversion
+// +kubebuilder:printcolumn:name="Address",type="string",JSONPath=".status.addresses[0].address",description="The VM address"
```

Comment:
Did some tool (kubebuilder?) make this and other whitespace changes to this file? None of them are directly related to your change.

Reply:
yeah i think this is kubebuilder that is doing this.